### PR TITLE
doc: summon.md: we now need gopass-summon-provider

### PR DESCRIPTION
**Pull request by marco-m**
RELEASE_NOTES=n/a

Signed-off-by: Marco Molteni <marco.molteni@mailbox.org>

### CODE REVIEW

**Comment by marco-m** [comment]
This closes #1633

File: docs/summon-provider.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ It is used to execute a process and inject secrets from a separate store. Using
 
 ## Summon Provider
 
-gopass can be used as [summon provider](https://cyberark.github.io/summon/#providers) out of the box, since it fulfills the summon provider contract.
+The gopass repository contains the [cmd/gopass-summon-provider](https://github.com/gopasspw/gopass/tree/master/cmd/gopass-summon-provider) tool, that can be used as [summon provider](https://cyberark.github.io/summon/#providers).
```
Comment (AnomalRoil) [inline docs/summon-provider.md:10]: Please use the [github relative links](https://github.blog/2013-01-31-relative-links-in-markup-files/) here.

**Review by AnomalRoil — CHANGES_REQUESTED** [review]
Also, if you are using Summon yourself, maybe you explain why it's interesting to use with Gopass? 
For example: will Summon erase the env var after usage or not?

**Review by dominikschulz — DISMISSED** [review]
LGTM. Thanks.

But please use relative links.

**Comment by marco-m** [comment]
Ah, it looks like that a force push dismisses reviews in this repo?
In any case, I updated the commit to use relative links.

> Also, if you are using Summon yourself, maybe you could explain why it's interesting to use with Gopass? I don't know anything about Summon myself, I think it could help users to have a clearer understanding of what is Summon and why it's nice to use with Gopass.

Maybe another time.

> For example: will Summon erase the env var after usage or not?

It would be impossible for summon to leave around the environment variables actually, due to how shells and subprocesses work.

In general, summon is well though-out from a security point of view.

**Review by dominikschulz — APPROVED** [review]
LGTM

**Pull request merged**